src/templates/index.twig

```diff
@@ -13,17 +13,17 @@
 
       <div class="row text-uppercase">
         <div class="col">
-          <h1 class="mb-3">Generic Hipster Coffee</h1>
-          <h2>Specialty coffee, fresh food &amp; local community.</h2>
-          <img alt="Generic hipster coffee area" class="img-fluid w-100 mt-4" src="/img/counter.jpg">
+          <h1>Generic Hipster Coffee</h1>
+          <h2>Specialty coffee,&nbsp;fresh food&nbsp;&amp;&nbsp;local community.</h2>
```

Comment:
&nbsp;fresh? not sure why

Reply:
Nevím, co tím myslíš? - fresh food je v návrhu

Reply:
nbsp mezi divnejma slovama